src/current/cockroachcloud/network-authorization.md

```diff
@@ -72,29 +76,6 @@ Refer to:
 - [Connect to a CockroachDB {{ site.data.products.serverless }} Cluster: Authorize your network]({% link cockroachcloud/connect-to-a-serverless-cluster.md %}#authorize-your-network).
 - [Connect to a CockroachDB {{ site.data.products.dedicated }} Cluster: Authorize your network]({% link cockroachcloud/connect-to-your-cluster.md %}#authorize-your-network).
 
-## VPC peering
```

Comment:
The most important bits of this section are moved to `connect-to-your-cluster.md`.